### PR TITLE
HOCS-3477 - FOI Approvals

### PR DESCRIPTION
**Pull request by agraham19**
* Add contributions style forms for Approvals

### CODE REVIEW

File: server/middleware/validation.js [hunk 1]
```diff
@@ -17,13 +17,38 @@ const validationErrors = {
...
+            });
+
+            if (result.length !== 0) {
+                valid = false;
+            }
+        } else {
+            valid = false;
+        }
+    } catch (error) {
+        valid = false;
```
Comment (gary-newbolt) [inline server/middleware/validation.js:46]: Any reason why we aren't logging out the error here? Seems to me the likely culprit would be malformed JSON response which is an error we would want to know about.

**Review by gary-newbolt — COMMENTED** [review]
Just that one thing on here. Thanks

File: server/middleware/validation.js (hunk 1 above)
Comment (agraham19) [reply server/middleware/validation.js:49]: I've now added a logging statement here.

Comment (gary-newbolt) [reply server/middleware/validation.js:49]: great thanks

**Review by gary-newbolt — APPROVED** [review]

**Pull request merged**